clang/test/CodeGenOpenCLCXX/array-type-infinite-loop.clcpp

```diff
@@ -0,0 +1,25 @@
+// NOTE: Assertions have been autogenerated by utils/update_cc_test_checks.py UTC_ARGS: --version 4
+//RUN: %clang_cc1 %s -emit-llvm -O1 -o - | FileCheck %s
```

Comment:
codegen tests need an explicit target

Reply:
add "triple spir", same as other tests in the same directory